server/src/main/scala/org/apache/livy/utils/SparkKubernetesApp.scala

```diff
@@ -553,8 +554,6 @@ private[utils] object KubernetesExtensions {
         .addNewPath()
         .withPath(s"/$appTag/?(.*)")
         .withNewBackend()
-        .withServiceName(service.getMetadata.getName)
```

Comment:
@mukeshkumarkulmi Should this read as follows to preserve the access 

```
        .withNewBackend()
        .withNewService()
        .withName(service.getMetadata.getName)
        .withPort(service.getSpec.getPorts.get(0).getName)
        .endBackend()

Reply:
![Screen Shot 2022-05-03 at 5 21 09 PM](https://user-images.githubusercontent.com/1307219/166606454-f9961e04-ed56-411c-a067-a7e4c247cc0d.png)

Also Tests need api changes

Reply:
![Screen Shot 2022-05-03 at 5 21 09 PM](https://user-images.githubusercontent.com/1307219/166606454-f9961e04-ed56-411c-a067-a7e4c247cc0d.png)

Also Tests need api changes